statistics/handle/update.go

```diff
@@ -208,6 +208,114 @@ func (h *Handle) NewSessionStatsCollector() *SessionStatsCollector {
 	return newCollector
 }
 
+// IndexUsageInformation is the data struct to store index usage information.
+type IndexUsageInformation struct {
+	QueryCount   int64
+	RowsSelected int64
+	LastUsedAt   string
+}
+
+type indexUsageMap map[string]IndexUsageInformation
+
+// SessionIndexUsageCollector is a list item that holds the index usage mapper. If you want to write or read mapper, you must lock it.
+type SessionIndexUsageCollector struct {
+	sync.Mutex
+
+	mapper  indexUsageMap
+	next    *SessionIndexUsageCollector
+	deleted bool
+}
+
+func (m indexUsageMap) updateByID(id string, value *IndexUsageInformation) {
```

Comment:
How about renaming it to `updateByKey(key string, value ...)`

Reply:
ok~